config/routes.rb

```diff
@@ -3,4 +3,13 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/studios", to: "studios#index"
+  get "studios/:id",to: "studios#show"
+  
+  get "/movies/:id", to: "movies#show"
+
+  post "/movie_actors", to: "movie_actors#create"
+
```

Comment:
These are great RESTful routes. I'm always weary about leaving the starting slash off of routes in Rails, it can sometimes lead to some unpredictable results. My rule is always to add it and it's one less bug you might have to deal with later.

The only other thing is the last route, although RESTful, it's not conventional. After we have out advanced routing class, we'd like to see this style of routing avoided.

```ruby
post "/movies/:id/actors", to: "movie_actors#create"
```